core/src/scene/sceneLoader.cpp

```diff
@@ -420,6 +414,7 @@ Node SceneLoader::propMerge(const std::string& propStr, const Mixes& mixes) {
     std::unordered_map<std::string, Mixes> mapMixes;
 
     for (const auto& mixNode: mixes) {
+        if (!mixNode.IsMap()) { continue; }
```

Comment:
Say someone wrote an empty style in the style sheet... before you start checking for map values in line 418, need to make sure this node is a Map node.